diff --git a/configure b/configure
index ecf92e7..bf51058 100755
--- a/configure
+++ b/configure
@@ -7815,8 +7815,6 @@ if eval test \${$as_ac_File+y}
 then :
   printf %s "(cached) " >&6
 else $as_nop
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
 if test -r "$LIBDIR"; then
   eval "$as_ac_File=yes"
 else
@@ -7855,8 +7853,6 @@ if eval test \${$as_ac_File+y}
 then :
   printf %s "(cached) " >&6
 else $as_nop
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
 if test -r "$INCDIR/cfitsio/fitsio.h"; then
   eval "$as_ac_File=yes"
 else
@@ -7878,8 +7874,6 @@ if eval test \${$as_ac_File+y}
 then :
   printf %s "(cached) " >&6
 else $as_nop
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
 if test -r "$INCDIR/fitsio.h"; then
   eval "$as_ac_File=yes"
 else
@@ -8074,8 +8068,6 @@ if eval test \${$as_ac_File+y}
 then :
   printf %s "(cached) " >&6
 else $as_nop
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
 if test -r "$INCDIR/pgplot/cpgplot.h"; then
   eval "$as_ac_File=yes"
 else
@@ -8097,8 +8089,6 @@ if eval test \${$as_ac_File+y}
 then :
   printf %s "(cached) " >&6
 else $as_nop
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
 if test -r "$INCDIR/cpgplot.h"; then
   eval "$as_ac_File=yes"
 else
