diff --git a/configure b/configure
index fa2fced..414a81a 100755
--- a/configure
+++ b/configure
@@ -7566,8 +7566,6 @@ $as_echo_n "checking for $LIBDIR... " >&6; }
 if eval \${$as_ac_File+:} false; then :
   $as_echo_n "(cached) " >&6
 else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
 if test -r "$LIBDIR"; then
   eval "$as_ac_File=yes"
 else
@@ -7604,8 +7602,6 @@ $as_echo_n "checking for $INCDIR/cfitsio/fitsio.h... " >&6; }
 if eval \${$as_ac_File+:} false; then :
   $as_echo_n "(cached) " >&6
 else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
 if test -r "$INCDIR/cfitsio/fitsio.h"; then
   eval "$as_ac_File=yes"
 else
@@ -7625,8 +7621,6 @@ $as_echo_n "checking for $INCDIR/fitsio.h... " >&6; }
 if eval \${$as_ac_File+:} false; then :
   $as_echo_n "(cached) " >&6
 else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
 if test -r "$INCDIR/fitsio.h"; then
   eval "$as_ac_File=yes"
 else
@@ -7819,8 +7813,6 @@ $as_echo_n "checking for $INCDIR/pgplot/cpgplot.h... " >&6; }
 if eval \${$as_ac_File+:} false; then :
   $as_echo_n "(cached) " >&6
 else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
 if test -r "$INCDIR/pgplot/cpgplot.h"; then
   eval "$as_ac_File=yes"
 else
@@ -7840,8 +7832,6 @@ $as_echo_n "checking for $INCDIR/cpgplot.h... " >&6; }
 if eval \${$as_ac_File+:} false; then :
   $as_echo_n "(cached) " >&6
 else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
 if test -r "$INCDIR/cpgplot.h"; then
   eval "$as_ac_File=yes"
 else
