diff --git a/pgsbox/pgtest.f b/pgsbox/pgtest.f
index f086f83..45e414f 100644
--- a/pgsbox/pgtest.f
+++ b/pgsbox/pgtest.f
@@ -57,7 +57,7 @@
       ESCAPE = CHAR(92)
 
 *     CALL PGBEG (0, '?', 1, 1)
-      call pgbeg (0, '/xw', 1, 1)
+      call pgbeg (0, '/null', 1, 1)
 
       CALL PGQINF ('TYPE', DEVTYP, J)
       IF (DEVTYP.EQ.'PS'  .OR.
diff --git a/pgsbox/cpgtest.c b/pgsbox/cpgtest.c
index b069e6a..12d7982 100644
--- a/pgsbox/cpgtest.c
+++ b/pgsbox/cpgtest.c
@@ -63,7 +63,7 @@ int main()
   trc[1] = naxis[1] + 0.5f;
 
   char devtyp[16];
-  strcpy(devtyp, "/XWINDOW");
+  strcpy(devtyp, "/NULL");
   cpgbeg(0, devtyp, 1, 1);
 
   int nchr = 16;
diff --git a/C/test/ttab3.c b/C/test/ttab3.c
index bb32387..280d8b3 100644
--- a/C/test/ttab3.c
+++ b/C/test/ttab3.c
@@ -71,7 +71,7 @@ int main()
 
 
   // PGPLOT initialization.
-  strcpy(text, "/xwindow");
+  strcpy(text, "/null");
   cpgbeg(0, text, 1, 1);
   cpgvstd();
   cpgsch(0.7f);
diff --git a/C/test/tcel2.c b/C/test/tcel2.c
index 97d68a0..ad27d88 100644
--- a/C/test/tcel2.c
+++ b/C/test/tcel2.c
@@ -103,7 +103,7 @@ int main()
 
 
   // PGPLOT initialization.
-  strcpy(text, "/xwindow");
+  strcpy(text, "/null");
   cpgbeg(0, text, 1, 1);
 
   // Define pen colours.
diff --git a/C/test/tpih2.c b/C/test/tpih2.c
index cd493cf..6241137 100644
--- a/C/test/tpih2.c
+++ b/C/test/tpih2.c
@@ -144,7 +144,7 @@ int main()
   trc[0] = naxis[0] + 0.5f;
   trc[1] = naxis[1] + 0.5f;
 
-  strcpy(devtyp, "/XWINDOW");
+  strcpy(devtyp, "/null");
   cpgbeg(0, devtyp, 1, 1);
   cpgvstd();
 
diff --git a/C/test/tspc.c b/C/test/tspc.c
index 1593354..a0b715d 100644
--- a/C/test/tspc.c
+++ b/C/test/tspc.c
@@ -67,7 +67,7 @@ int main()
 
   // PGPLOT initialization.
   char text[80];
-  strcpy(text, "/xwindow");
+  strcpy(text, "/null");
   cpgbeg(0, text, 1, 1);
 
   int    naxisj = NSPEC;
diff --git a/C/test/tcel1.c b/C/test/tcel1.c
index 70d90bd..f376b52 100644
--- a/C/test/tcel1.c
+++ b/C/test/tcel1.c
@@ -77,7 +77,7 @@ int main()
 
 
   // PGPLOT initialization.
-  strcpy(text, "/xwindow");
+  strcpy(text, "/null");
   cpgbeg(0, text, 1, 1);
 
   // Define pen colours.
diff --git a/C/test/ttab2.c b/C/test/ttab2.c
index 3d36a69..7e26f99 100644
--- a/C/test/ttab2.c
+++ b/C/test/ttab2.c
@@ -74,7 +74,7 @@ int main()
 
 
   // PGPLOT initialization.
-  strcpy(text, "/xwindow");
+  strcpy(text, "/null");
   cpgbeg(0, text, 1, 1);
   cpgvstd();
   cpgsch(0.7f);
diff --git a/C/test/tprj2.c b/C/test/tprj2.c
index 190fcc0..dd2b0a9 100644
--- a/C/test/tprj2.c
+++ b/C/test/tprj2.c
@@ -62,7 +62,7 @@ int main()
 
 
   // PGPLOT initialization.
-  strcpy(text, "/xwindow");
+  strcpy(text, "/null");
   cpgbeg(0, text, 1, 1);
 
   // Define pen colours.
diff --git a/C/test/twcsmix.c b/C/test/twcsmix.c
index b2a3ba2..0e63248 100644
--- a/C/test/twcsmix.c
+++ b/C/test/twcsmix.c
@@ -90,7 +90,7 @@ int main()
 
 
   // PGPLOT initialization.
-  strcpy(text, "/xwindow");
+  strcpy(text, "/null");
   cpgbeg(0, text, 1, 1);
 
   // Define pen colours.
diff --git a/Fortran/test/ttab3.f b/Fortran/test/ttab3.f
index b613aec..e909d49 100644
--- a/Fortran/test/ttab3.f
+++ b/Fortran/test/ttab3.f
@@ -65,7 +65,7 @@
 
 
 *     PGPLOT initialization.
-      CALL PGBEG (0, '/xwindow', 1, 1)
+      CALL PGBEG (0, '/null', 1, 1)
       CALL PGVSTD ()
       CALL PGSCH (0.7)
       CALL PGWNAD (-135.0, 135.0, -95.0, 140.0)
diff --git a/Fortran/test/tspc.f b/Fortran/test/tspc.f
index fc27ba2..e85cc73 100644
--- a/Fortran/test/tspc.f
+++ b/Fortran/test/tspc.f
@@ -53,7 +53,7 @@
       NFAIL = 0
 
 *     PGPLOT initialization.
-      CALL PGBEG (0, '/xwindow', 1, 1)
+      CALL PGBEG (0, '/null', 1, 1)
 
       NAXISJ = NSPEC
       CRPIXJ = NAXISJ/2 + 1
diff --git a/Fortran/test/tcel1.f b/Fortran/test/tcel1.f
index 653f294..cb693b4 100644
--- a/Fortran/test/tcel1.f
+++ b/Fortran/test/tcel1.f
@@ -75,7 +75,7 @@
 
 
 *     PGPLOT initialization.
-      CALL PGBEG (0, '/xwindow', 1, 1)
+      CALL PGBEG (0, '/null', 1, 1)
 
 *     Define pen colours.
       CALL PGSCR (0, 0.00, 0.00, 0.00)
diff --git a/Fortran/test/tpih2.f b/Fortran/test/tpih2.f
index c51ca97..f6f1a08 100644
--- a/Fortran/test/tpih2.f
+++ b/Fortran/test/tpih2.f
@@ -129,7 +129,7 @@
       TRC(1) = NAXIS(1) + 0.5
       TRC(2) = NAXIS(2) + 0.5
 
-      DEVTYP = '/XWINDOW'
+      DEVTYP = '/null'
       CALL PGBEG(0, DEVTYP, 1, 1)
       CALL PGVSTD()
 
diff --git a/Fortran/test/tprj2.f b/Fortran/test/tprj2.f
index 37ebec0..fc31cc2 100644
--- a/Fortran/test/tprj2.f
+++ b/Fortran/test/tprj2.f
@@ -46,7 +46,7 @@
  20   CONTINUE
 
 *     PGPLOT initialization.
-      CALL PGBEG (0, '/xwindow', 1, 1)
+      CALL PGBEG (0, '/null', 1, 1)
 
 *     Define pen colours.
       CALL PGSCR (0, 0.00, 0.00, 0.00)
diff --git a/Fortran/test/twcsmix.f b/Fortran/test/twcsmix.f
index 0cc2332..dfa84fb 100644
--- a/Fortran/test/twcsmix.f
+++ b/Fortran/test/twcsmix.f
@@ -87,7 +87,7 @@
      :        '-----------------------------------------')
 
 *     PGPLOT initialization.
-      CALL PGBEG (0, '/xwindow', 1, 1)
+      CALL PGBEG (0, '/null', 1, 1)
 
 *     Define pen colours.
       CALL PGSCR (0, 0.00, 0.00, 0.00)
diff --git a/Fortran/test/ttab2.f b/Fortran/test/ttab2.f
index a47a96a..b3a1758 100644
--- a/Fortran/test/ttab2.f
+++ b/Fortran/test/ttab2.f
@@ -70,7 +70,7 @@
 
 
 *     PGPLOT initialization.
-      CALL PGBEG (0, '/xwindow', 1, 1)
+      CALL PGBEG (0, '/null', 1, 1)
       CALL PGVSTD ()
       CALL PGSCH (0.7)
 
